specification/app/data-plane/Microsoft.App.DynamicSessions/preview/2024-10-02-preview/examples/SessionResourceFiles_UploadFile.json

```diff
@@ -0,0 +1,21 @@
+{
+  "title": "SessionResourceFiles_UploadFile",
+  "operationId": "SessionResourceFiles_UploadFile",
+  "parameters": {
+    "api-version": "2024-10-02-preview",
+    "identifier": "testidentidier",
```

Comment:
NIT; can we rename this to `testIdentifier`

Reply:
Yes, thanks, fixed